samples/attested_tls/AttestedTLSREADME.md

```diff
@@ -24,7 +24,7 @@ There are two types of Attested TLS connections:
 
   - [Transport Layer Security (TLS)](https://en.wikipedia.org/wiki/Transport_Layer_Security) a cryptographic protocol designed to provide communications security over a computer network.
 
-  - [Open Enclave Remote Attestation](https://github.com/openenclave/openenclave/tree/master/samples/remote_attestation#what-is-attestation): Remote Attestation is the concept of a HW entity or of a
+  - [Open Enclave Remote Attestation](https://github.com/openenclave/openenclave/tree/master/samples/attestation#what-is-attestation): Remote Attestation is the concept of a HW entity or of a
```

Comment:
The attestation sample works for both local and remote attestation. Does the Attested TLS sample work for both types attestation? If so how about drop  all reference of "remote" from the text?

Reply:
Attested TLS targets on remote attestation.